src/operator/pairwise.ts

```diff
@@ -8,12 +8,13 @@ import {Subscriber} from '../Subscriber';
  * The (N-1)th is stored in the internal state until Nth input occurs.
  * @returns {Observable<R>} an observable of pairs of values.
  */
-export function pairwise<T>(): Observable<T> {
-  return this.lift(new PairwiseOperator());
+export function pairwise<T>(): Observable<[T, T]> {
+  let _this: Observable<T> = this;
```

Comment:
@luisgabriel The idea is that using `this` gives you `any`, and then all type checking from that point on is disabled. Using only `_this` ensures your type signatures are compatible (i.e. lifting through the operator actually produces the output type of the function). We can also do this using casts, but I arrived at this pattern over the course of a number of changes across other operators.